docs/app/Examples/addons/Confirm/Variations/ConfirmExampleSize.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+import { Button, Confirm } from 'semantic-ui-react'
+
+class ConfirmExampleSize extends Component {
```

Comment:
Let's use a in definition export, i.e. `export default class ConfirmExampleSize`